src/commands/playercount.ts

```diff
@@ -17,7 +17,25 @@ const playerCount = async (options: CommandExecuteParameters) => {
     return isSlash ? interaction?.reply(response) : message?.reply(response)
   }
   // positive number, give more details
-  const response = `There are currently **${playerCount.Online_players} operatives online.** \n There are **${playerCount.Open_lobbies} open lobbies** and **${playerCount.Active_games} active games.**`
+
+  // Sets default text strings.
+  var operativesText = `There are currently **${playerCount.Online_players} operatives online.**`;
+  var lobbiesText = `There are **${playerCount.Open_lobbies} open lobbies**`;
+  var activeGamesText = `games`;
```

Comment:
I believe this hould be 
```  var activeGamesText = ` ${playerCount.Active_games} games`;```

Reply:
See [Here](https://github.com/Mastagoon/untrusted-discord/pull/8/commits/e595efb7f12d09b06d92d66645ac3505726d512b) or commit 3.